### PR TITLE
Add LDAP Tracing Panel to the third-party list

### PR DESCRIPTION
**Pull request by danyi1212**
LDAP Operations Tracing panel from django-windowsauth module.

GitHub: https://github.com/danyi1212/django-windowsauth
Docs: https://django-windowsauth.readthedocs.io/en/latest/howto/debug_toolbar.html

### CODE REVIEW

File: docs/panels.rst [hunk 1]
```diff
@@ -189,6 +189,17 @@ line_profiler_.
 
 .. _line_profiler: https://github.com/rkern/line_profiler
 
+LDAP Tracing
```
Comment (tim-schilling) [inline docs/panels.rst:192]: Can you move this entire section above `Line Profiler` so it's alphabetical?

File: docs/panels.rst [hunk 2]
```diff
@@ -189,6 +189,17 @@ line_profiler_.
...
+~~~~~~~~~~~~
+
+URL: https://github.com/danyi1212/django-windowsauth
+
+Path: ``windows_auth.panels.LDAPPanel``
+
+LDAP Operations performed during the request, including timing, request and response messages, the entries recieved, write changes list, stacktracing and error debugging.
```
Comment (tim-schilling) [inline docs/panels.rst:199]: This is one of the lines causing the lint test to fail. Try running `make style` or shorten the lines here. Line 201 may also need it.

Please change recieved to received.

**Review by tim-schilling — CHANGES_REQUESTED** [review]
Thanks for the PR. I have a few nitpicks before merging.

File: docs/panels.rst (hunk 2 above)
Comment (danyi1212) [reply docs/panels.rst]: Oh totally missed that 😅 Thank you!

File: docs/panels.rst (hunk 1 above)
Comment (danyi1212) [reply docs/panels.rst]: Fixed that! Thank you

**Review by tim-schilling — APPROVED** [review]
Thank you for making those changes and for the PR! I'll merge this shortly.

**Pull request merged**